src/components/smartSearch/FilterPreview.tsx

```diff
@@ -0,0 +1,86 @@
+    SurveyResponseFilterConfig, SurveySubmissionFilterConfig, UserFilterConfig } from 'types/smartSearch';
+
+interface FilterPreviewProps {
+    filter: SmartSearchFilterWithId;
+    onDeleteFilter: (filter: SmartSearchFilterWithId) => void;
+    onEditFilter: (filter: SmartSearchFilterWithId) => void;
+}
+
+const FilterPreview = ({ filter, onDeleteFilter, onEditFilter }:FilterPreviewProps): JSX.Element => {
+    return (
+        <Box display="flex" width={ 1 }>
+            <Box alignItems="center" display="flex" flex={ 20 } justifyContent="center">
+                <Typography align="center" variant="body2">
+                    { filter.type === FILTER_TYPE.ALL &&
+                    <DisplayAll /> }
+                    { filter.type === FILTER_TYPE.CALL_HISTORY &&
+                    <DisplayCallHistory
+                        filter={ filter as SmartSearchFilterWithId<CallHistoryFilterConfig> }
+                    /> }
+                    { filter.type === FILTER_TYPE.CAMPAIGN_PARTICIPATION &&
+                    <DisplayCampaignParticipation
+                        filter={ filter as SmartSearchFilterWithId<CampaignParticipationConfig> }
+                    /> }
+                    { filter.type === FILTER_TYPE.MOST_ACTIVE &&
+                    <DisplayMostActive
+                        filter={ filter as SmartSearchFilterWithId<MostActiveFilterConfig>  }
+                    /> }
+                    { filter.type === FILTER_TYPE.PERSON_DATA &&
+                    <DisplayPersonData
+                        filter={ filter as SmartSearchFilterWithId<PersonDataFilterConfig>  }
+                    /> }
+                    { filter.type === FILTER_TYPE.PERSON_TAGS &&
+                    <DisplayPersonTags
+                        filter={ filter as SmartSearchFilterWithId<PersonTagsFilterConfig>  }
+                    /> }
+                    { filter.type === FILTER_TYPE.RANDOM &&
+                    <DisplayRandom
+                        filter={ filter as SmartSearchFilterWithId<RandomFilterConfig>  }
+                    /> }
+                    { filter.type === FILTER_TYPE.SURVEY_RESPONSE &&
+                    <DisplaySurveyResponse
+                        filter={ filter as SmartSearchFilterWithId<SurveyResponseFilterConfig>  }
+                    /> }
+                    { filter.type === FILTER_TYPE.SURVEY_SUBMISSION &&
+                    <DisplaySurveySubmission
+                        filter={ filter as SmartSearchFilterWithId<SurveySubmissionFilterConfig>  }
+                    /> }
+                    { filter.type === FILTER_TYPE.USER &&
+                    <DisplayUser
+                        filter={ filter as SmartSearchFilterWithId<UserFilterConfig>  }
+                    /> }
+                </Typography>
+            </Box>
+            <Box flex={ 1 }>
+                <IconButton
+                    onClick={ () => onEditFilter(filter) }>
+                    <Settings />
```

Comment:
What do you think about using a pen icon instead of the cogwheel?

Reply:
I always hated the look of the pen and aren't we using cogwheels elsewhere? I vote for cog for consistency (for now)

Reply:
Ok we can keep the cogwheel, but when we use it elsewhere we use it to open a menu, not like this. So it's not really consistent to use it as a link to another screen. But I don't feel strongly about this. 😊